content/home/mission.md

```diff
@@ -15,7 +15,7 @@ We make interactive computing more accessible and powerful for research and educ
 <div class="card-group mission-cards">
     <div class="card w-50 m-3" style="width: 18rem;">
         <div class="card-body text-center">
-            <h5 class="card-title">Manage interactive computing infrastructure for research and education</h5>
+            <h5 class="card-title">Manage and build interactive computing infrastructure for research and education</h5>
             <p class="card-text">
             2i2c provides managed JupyterHubs for data science in research and engineering communities. They are tailored for the communities they serve and 💯 open source.
```

Comment:
Not part of the diff, but I'd suggest using "offers" instead of "provides" here. The idea is that we offer something that others must decide to accept and purchase. To me, "provides" sounds like something that's always available, whereas I think our offerings have more of a "product" flavor to them...

Ignore if it's my non-native English brain misreading though...